force-app/main/default/pages/StudentList.page-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexPage xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <availableInTouch>false</availableInTouch>
+    <confirmationTokenRequired>false</confirmationTokenRequired>
+    <label>StudentList</label>
+    <packageVersions>
```

Comment:
Was this included as part of a package?

Reply:
<packageVersions> were removed from the current branch.